### ISSUE TITLE
mongoose connection

### ISSUE THREAD

**Issue by anand-paragon** [sushant8421/portfolio_dev#3]
Do not use hard coded value keep it in config.json or environment variable
**suggestion** -  use `dotenv` npm package for configuration management

- If unable to connect to database application should not continue it should either retry or exit from process `process.exit`

https://github.com/sushant8421/portfolio_dev/blob/26a3283ba197d6cd16987a4ba882f4ba65e598ac/index.js#L18

**Comment by sushant8421** [comment]
yeah i learned that also, but it was just for checking purpose and i will modify it as you suggested.
